client/my-sites/site-settings/form-seo.jsx

```diff
@@ -400,8 +406,10 @@ export const SeoForm = React.createClass( {
 					) }
 				</Card>
 
+				{ ! showAdvancedSeo && <SeoSettingsUpgradeNudge { ...{ upgradeToBusiness } } /> }
```

Comment:
Should we check for `! hasBusinessPlan` here instead? Otherwise, if feature flag is disabled, upgrade nudge will be shown to users that already have a business plan.